rules/swrl_rules.py

```diff
@@ -0,0 +1,160 @@
+import json
+from owlready2 import sync_reasoner_pellet, World, Ontology, Imp, Nothing
+import logging
+from rdflib import Graph, Namespace
+
+
```

Comment:
Please add docstring to these functions. It is not clear why the first ontology in the list is an ontology network and the remaining are ontology locations.

Reply:
added docstrings, see [line 9](https://github.com/VCityTeam/UD-Graph/blob/aa846ff115fd7ce7eb5a3348a11464d4a3b35351/rules/swrl_rules.py#L9) about the ontology network. Ontology networks don't exist in OWLReady2 but you can import multiple ontologies into an existing ontology which (assuming alignments exist) is a network.

Alternatively, we can change this variable name to just ontology to be clearer